networkit/cpp/algebraic/test/MatricesGTest.cpp

```diff
@@ -120,11 +120,26 @@ class MatricesGTest : public testing::Test {
     template <class Matrix>
     void testParallelForNonZeroElementsInRowOrder();
 
+    template <class Matrix>
+    void testMatrixToGraph();
+
     // TODO: Test other matrix classes
 
     // TODO: Test mmT multiplication, etc.!
 };
 
+template <class Matrix>
+void MatricesGTest::testMatrixToGraph() { 
+    
+    METISGraphReader graphReader;
+    Matrix mat = Matrix::adjacencyMatrix(graphReader.read("input/power.graph"));
+    Graph G =MatrixTools::matrixToGraph(mat);
+
+    ASSERT_EQ(G.numberOfEdges(),6594);
```

Comment:
```suggestion
    EXPECT_EQ(G.numberOfEdges(), 6594);
```
See http://google.github.io/googletest/reference/assertions.html